packages/minimongo/local_collection.js

```diff
@@ -1200,7 +1200,7 @@ LocalCollection._selectorIsIdPerhapsAsObject = selector => {
   if (LocalCollection._selectorIsId(selector))
     return true;
 
-  return LocalCollection._selectorIsId(selector && selector._id) && Object.keys(selector).length === 1;
```

Comment:
Before, this function would only return true when `_id` was the only property in the selector, but now it'll return true when the `_id` property in the selector is an ID, regardless of whether or not other properties are set.

Looking at usages of this function there could be some cases where the following query is now considered an `_id`-only query:
```
{
    "_id": "foo",
    "bar": "baz"
}
```
For instance in [allow-deny](https://github.com/meteor/meteor/blob/8005532f4ba847d3e8bb4c1100d09bf53136ff0a/packages/allow-deny/allow-deny.js#L265), where updates are denied when it's not a pure `_id` query. In [minimongo](https://github.com/meteor/meteor/blob/08f5ea4c2b75c003df6625ed5e9dadea02e4414d/packages/minimongo/minimongo.js#L98) it looks like this change would return the document with id "foo" regardless of whether the "bar" part matches.